packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -98,11 +105,15 @@ export default class LocusInfo {
   /**
    * updates the locus with full locus object
    * @param {object} locus locus object
+   * @param {sting} eventType particulat locus event
    * @returns {object} null
    */
-  onFullLocus(locus) {
+  onFullLocus(locus, eventType) {
+    console.log('EVENTS', eventType);
```

Comment:
are the extraneous console.logs necessary? we will have to remove them anyway when we do the logging update

Reply:
removed just added that for logging for call ended event

Reply:
its just the type of event which came in . We can do clean up when we add logger module

Reply:
ok